spec/models/activity_spec.rb

```diff
@@ -1,5 +1,11 @@
 require 'rails_helper'
 
 describe Activity, type: :model do
+  let(:story) { create(:story, :with_project, :with_activity) }
 
+  describe ".by_story" do
+    subject { Activity.by_story(story.id).map(&:subject_type) }
+
+    it { is_expected.to contain_exactly('Story') }
```

Comment:
Could you improve your test cases?
Your test just expect `Activity.by_story(story_id)` to bring a `subject_type` equals to `Story` (what is hardcoded in the scope).

I think a better spec is to check if you can find the story by id.

```Ruby
describe Activity, type: :model do
  describe ".by_story" do
    let(:resource_id) { 1001 }

    before do
      create_list(:story, 3, title: 'not_found')
      create(:activity, id: resource_id)
    end

    subject { Activity.by_story(story_id) }

    context "when the story id is found" do
      let(:story)    { create(:story, :with_activity, id: resource_id, title: 'found_by_id') }
      let(:story_id) { story.id }

      it "returns the story where the id matches" do
        expect(subject.map(&:title)).to contain_exactly('found_by_id')
      end

      it "returns only the 'Story' subject type" do
        expect(subject.map(&:subject_type)).to contain_exactly('Story')
      end
    end

    context "when the story id is not found" do
      let(:story_id) { nil }

      it "returns an empty collection" do
        is_expected.to be_empty
      end
    end
  end
end
```